module/analytics/src/commonMain/kotlin/org/cru/godtools/shared/analytics/AnalyticsScreenNames.kt

```diff
@@ -0,0 +1,27 @@
+package org.cru.godtools.shared.analytics
+
+object AnalyticsScreenNames {
+    // region Dashboard Screens
+    const val DASHBOARD_HOME = "Favorites"
+    const val DASHBOARD_ALL_TOOLS = "All Tools"
+    const val DASHBOARD_LESSONS = "Lessons"
+    // endregion Dashboard Screens
+
+    // region Account
+    const val ACCOUNT_ACTIVITY = "Personal Dashboard"
```

Comment:
@levieggertcru

Reply:
it'll be accessible on iOS using: `AnalyticsScreenNames.shared.ACCOUNT_ACTIVITY`